cluster/saltbase/salt/kubelet/default

```diff
@@ -164,6 +164,11 @@
   {% set hairpin_mode = "--hairpin-mode=" + pillar['hairpin_mode'] -%}
 {% endif -%}
 
+{% set babysit_daemons = "" -%}
```

Comment:
nit: would be better if we allowed everyone else to pass this through as an env var. Now any provider that's not aws/gce is forced to set it to false. This can come in a follow up pr if you want to do that.